jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -209,16 +240,19 @@ The CLI attempts to autodetect whether to split by filename or classname based o
 cat my_java_test_classnames | circleci tests split --split-by=timings --timings-type=classname
 ```
 
-For tests with missing timing data, a random small timing value is assigned. You can override this default value with the `--time-default` flag:
+##### Set the default value for missing timing data
+{: #set-the-default-value-for-missing-timing-data }
+
+For partially found test results, any tests with missing data is assigned a random small value. You can override this default value with the `--time-default` flag:
```

Comment:
> any tests with missing data is

This needs to be either:
- ...any tests with missing data are...
or
- ...any test with missing data is...